lib/project_types/extension/tasks/execute_commands/create.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+require "shopify_cli"
+
+module Extension
+  module Tasks
+    module ExecuteCommands
+      class Create < Base
+        property!  :template, accepts: Models::ServerConfig::Development::VALID_TEMPLATES
+        property!  :root_dir, accepts: String
```

Comment:
What does `root_dir` represent? Is it the root of the `extension`?

Reply:
yes, it represents the root of the extension and it is the term also used in the extensions server